test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,43 @@
+require "test_helper"
+
+describe RecipesController do
+
+  describe 'index' do
+
+    it "can get a list of recipes" do
+      VCR.use_cassette('list_recipes') do
+        get recipes_path, params: { q: "chicken"}
+
+        must_respond_with :ok
+      end
+    end
+
+    it 'will flash warning if search results in 0 entries found' do
```

Comment:
There are some interesting cases you're not covering here:

- What happens if no search term is provided?
- What if the search returns no results?

It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?